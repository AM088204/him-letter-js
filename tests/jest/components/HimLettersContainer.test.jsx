import React from 'react';
import HimLettersContainer from '../../../src/him-letters-js/components/HimLettersContainer';

it('should render a HimLettersContainer with a default HimLettersView', () => {
  const testContainer = <HimLettersContainer />;
  expect(shallow(testContainer)).toMatchSnapshot();
});
