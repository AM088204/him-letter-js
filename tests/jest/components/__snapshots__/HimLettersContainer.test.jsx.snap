// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a HimLettersContainer with a default HimLettersView 1`] = `
<ApplicationLoadingOverlay
  backgroundStyle="clear"
  isOpen={true}
/>
`;
